Dockerfile

```diff
@@ -10,55 +10,48 @@
     export MOCK_API=true && \
     go test -mod=vendor -v ./... && \
     GOOS=linux GOARCH=${ARCH} CGO_ENABLED=0 go build -mod=vendor -o /tmp/che-operator/che-operator cmd/manager/main.go
 
-# upstream, download devworkspace-operator templates for every build
-# downstream, copy prefetched zip into /tmp
-RUN curl -L https://api.github.com/repos/devfile/devworkspace-operator/zipball/${DEV_WORKSPACE_CONTROLLER_VERSION} > /tmp/devworkspace-operator.zip && \
-    unzip /tmp/devworkspace-operator.zip */deploy/deployment/* -d /tmp && \
+RUN unzip /tmp/asset-devworkspace-operator.zip */deploy/deployment/* -d /tmp && \
     mkdir -p /tmp/devworkspace-operator/templates/ && \
     mv /tmp/devfile-devworkspace-operator-*/deploy /tmp/devworkspace-operator/templates/
 
-# upstream, download devworkspace-che-operator templates for every build
-# downstream, copy prefetched zip into /tmp
-RUN curl -L https://api.github.com/repos/che-incubator/devworkspace-che-operator/zipball/${DEV_WORKSPACE_CHE_OPERATOR_VERSION} > /tmp/devworkspace-che-operator.zip && \
-    unzip /tmp/devworkspace-che-operator.zip */deploy/deployment/* -d /tmp && \
+RUN unzip /tmp/asset-devworkspace-che-operator.zip */deploy/deployment/* -d /tmp && \
     mkdir -p /tmp/devworkspace-che-operator/templates/ && \
     mv /tmp/che-incubator-devworkspace-che-operator-*/deploy /tmp/devworkspace-che-operator/templates/
 
 # Build restic. Needed for backup / restore capabilities
-ENV RESTIC_TAG=v0.12.0
 RUN mkdir -p $GOPATH/restic && cd $GOPATH/restic && \
-    curl -L https://api.github.com/repos/restic/restic/zipball/${RESTIC_TAG} > /tmp/restic.zip && \
-    unzip /tmp/restic.zip -d /tmp && \
+    unzip /tmp/asset-restic.zip -d /tmp && \
     mv /tmp/restic-*/* $GOPATH/restic && \
     export ARCH="$(uname -m)" && if [[ ${ARCH} == "x86_64" ]]; then export ARCH="amd64"; elif [[ ${ARCH} == "aarch64" ]]; then export ARCH="arm64"; fi && \
     go mod vendor && \
     GOOS=linux GOARCH=${ARCH} CGO_ENABLED=0 go build -mod=vendor -o /tmp/restic/restic ./cmd/restic
 
 # https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/ubi8-minimal
-FROM registry.access.redhat.com/ubi8-minimal:8.4-200.1622548483
+FROM registry.access.redhat.com/ubi8-minimal:8.4-205
 
 COPY --from=builder /tmp/che-operator/che-operator /usr/local/bin/che-operator
-COPY --from=builder /che-operator/templates/keycloak-provision.sh /tmp/keycloak-provision.sh
-COPY --from=builder /che-operator/templates/keycloak-update.sh /tmp/keycloak-update.sh
-COPY --from=builder /che-operator/templates/oauth-provision.sh /tmp/oauth-provision.sh
-COPY --from=builder /che-operator/templates/delete-identity-provider.sh /tmp/delete-identity-provider.sh
-COPY --from=builder /che-operator/templates/create-github-identity-provider.sh /tmp/create-github-identity-provider.sh
-
+COPY --from=builder /che-operator/templates/*.sh /tmp/
```

Comment:
Can't we just copy everything from templates?

Reply:
we could but right now there are only 5 files in that folder:
```
create-github-identity-provider.sh
delete-identity-provider.sh
keycloak-provision.sh
keycloak-update.sh
oauth-provision.sh
```

:D